src/Airport.js

```diff
@@ -0,0 +1,49 @@
+const Plane = require('./Plane');
+
+class Airport {
+
+    planesAtAirport = [];
+
+    constructor(airportCapacity = 10) {
+        this.airportCapacity = airportCapacity;
+    }
+
+    landPlane(plane, weather) {
+        if (weather === `sunny` && plane instanceof Plane) {
+            if (!this.isFull() && !this.isPlaneAtAirport(plane.getID())) {
+                this.planesAtAirport.push(plane)
+            }
+        }
+    }
+
+    planeTakeOff(plane, weather) {
+        if (this.isPlaneAtAirport(plane.getID()) && weather === `sunny`) {
+            const index = this.planesAtAirport.indexOf(plane);
+            if (index > -1) {
+                this.planesAtAirport.splice(index, 1);
+            }
+        }
+    }
+
+    isPlaneAtAirport(planeID) {
+        for (let i = 0; i < this.planesAtAirport.length; i++) {
+            if (this.planesAtAirport[i].getID() === planeID) {
+                return true;
```

Comment:
**isPlaneAtAirport Method**: I feel like I could have refactored the loop, which I did attempt, but was not successful, so need to study more on possible loop options